CONTRIBUTING.md

```diff
@@ -9,7 +9,7 @@ Before anything else, thank you. Thank you for taking some of your precious time
 
 ## Tests
 
-See the `package.json` scripts section for how to run each category of tests.
+Now type `yarn run` or see the `package.json` scripts section for how to run each category of tests.
```

Comment:
```suggestion
```
This is not related to the PR, and another PR suggested the same change

Reply:
Yeah, I forgot that I had that other change on the same machine and we branched this PR on top of it.

Reply:
We should merge that change in and rebase this branch. I don't know why, but the build has been stuck on that other branch.